core/src/main/java/nl/nn/adapterframework/doc/model/GenericRole.java

```diff
@@ -0,0 +1,181 @@
+package nl.nn.adapterframework.doc.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.Logger;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import nl.nn.adapterframework.doc.Utils;
+import nl.nn.adapterframework.util.LogUtil;
+
+public class GenericRole {
```

Comment:
Ook hier graag een javadoc beschrijving van wat een GenericRole is, volgens mij: de verzameling ElementRoles met dezelfde syntax1naam, die voor komen als ConfigChildren van een FrankElement